src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -384,18 +387,23 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                             let zAxisMultiplier = this._scene.useRightHandedSystem? 1 : -1;
                             const indexTipPos = indexTipPose.transform.position;
                             const indexTipOrientation = indexTipPose.transform.orientation;
+                            controllerData.indexTipQuaternion.set(indexTipOrientation.x, indexTipOrientation.y, (indexTipOrientation.z * zAxisMultiplier), (indexTipOrientation.w * zAxisMultiplier));
+
                             // set positions for near pick and hover
                             if (controllerData.pickIndexMeshTip) {
                                 controllerData.pickIndexMeshTip.position.set(indexTipPos.x, indexTipPos.y, (indexTipPos.z * zAxisMultiplier));
                             }
                             if (controllerData.hoverIndexMeshTip) {
-                                controllerData.hoverIndexMeshTip.position.set(indexTipPos.x, indexTipPos.y, (indexTipPos.z * zAxisMultiplier));

```

Comment:
Why was this removed?

Reply:
i think you are viewing outdated file for some reason